Core/src/IfcGeometryConverter/SplineConverter.h

```diff
@@ -43,19 +47,27 @@ namespace OpenInfraPlatform
 				public:
 					SplineConverterT(
 						std::shared_ptr<GeometrySettings> geomSettings,
-						std::shared_ptr<UnitConverter<IfcEntityTypesT>> unitConverter,
-						std::shared_ptr<PlacementConverterT<IfcEntityTypesT>> pc)
+						std::shared_ptr<UnitConverter<IfcEntityTypesT>> unitConverter)
 						:
-						ConverterBaseT<IfcEntityTypesT>(geomSettings, unitConverter),
-						placementConverter(pc)
+						ConverterBaseT<IfcEntityTypesT>(geomSettings, unitConverter) 
 					{}
 
 					virtual ~SplineConverterT() {}
 
+					/*! \brief Converts \c IfcBSplineCurve subtypes to a vector of curve points, which can be print on the screen.
+					 *
+					 * This converter can handle
+					 * \c IfcBSplineCurveWithKnots and
+					 * \c IfcRationalBSplineCurveWithKnots, which are subtypes of IfcBSplineCurve.
+					 *
+					 * \param[in]	splineCurve		\c IfcBSplineCurve entity to be converted.
+					 * \param[in]	controlPoints	A vector of the B-Spline control points, must be obtain from the \c IfcBSplineCurve entity.
+					 * \param[out]	loops			The vector of curve points, which can be print on the screen.
+					 */
 					void convertIfcBSplineCurve(
 						const EXPRESSReference<typename IfcEntityTypesT::IfcBSplineCurve>& splineCurve,
 						const std::vector<carve::geom::vector<3>>& controlPoints,
-						std::vector<carve::geom::vector<3>>& loops)
+						std::vector<carve::geom::vector<3>>& loops) throw(...)

```

Comment:
```suggestion
						std::vector<carve::geom::vector<3>>& loops) const throw(...)
```
The function does not change the state of the SplineConverter class' instance, therefore it can be made `const`.

Reply:
This can probably be applied to all functions of the class.

Reply:
Done in commit 27331ec